README.md

```diff
@@ -17,14 +30,33 @@ To activate the virtual environment then run
 ```
 poetry shell
 ```
-To add a new core library package then run the command
-```
-poetry add <package>
-```
-To add a new developer library patckage then run the command
-```
-poetry add --group dev <package>
+
+## Usage 🚀
+(Deployment on Streamlit Cloud is coming)
+
+To run the GPTravel web app on your local machine, use the following command:
 ```
-To remove a package then run the command
+streamlit run prototype.py
 ```
-poetry remove <package>
\ No newline at end of file
+This will start the GPTravel app using Streamlit. You can then access the web app through your browser.
+
+## Next Steps 🌟
+Here are some suggested next steps to enhance GPTravel:
+
+* Creation of a Travel Score based on the diversity of the activities generated by GPTravel.
```

Comment:
@RobertoCorti  I do not understand this point. We have implemented a metric based on entropy information to provide a score to the variety of the travel plan activities. Maybe do you mean something else?

Reply:
Yes, maybe I should remove it...